django_netjsongraph/base/admin.py

```diff
@@ -23,6 +23,19 @@ def __init__(self, *args, **kwargs):
 class BaseAdmin(TimeStampedEditableAdmin):
     save_on_top = True
 
+    def get_extra_context(self):
+        ctx = {'submit_line': 'admin/django_netjsongraph/submit_line.html'}
+        return ctx
```

Comment:
simplify to: `return {'submit_line': 'admin/django_netjsongraph/submit_line.html'}`